addons/udes_stock/models/stock_inventory.py

```diff
@@ -84,12 +104,41 @@ def _is_adjusting_reserved(self):
                 return True
         return False
 
+    def _has_theoretical_quantity_changed(self):
+        """Check if any theoretical quantity in the location has changed.
+        """
+        self.ensure_one()
+        for line in self.line_ids:
```

Comment:
May need to override this in some customer repos e.g. to cover off quants with GRNs and ODNs

Reply:
Yeah have added a comment here: https://github.com/unipartdigital/udes-open/pull/454#discussion_r577669955

Think it is worth refactoring into its own function to make it easier to add in/take out fields to display as needed.

Reply:
This function should be ok as McLaren overwrites the function `_get_quants()` to return quants matching grn/odn

Reply:
Yep sorry got a bit tripped up there. We refactored the stock module to make it more easily overridable so that shouldn't be an issue.

I thought this comment was also referring to the message on the popup view, my bad. :)